src/App/App.tsx

```diff
@@ -24,20 +27,23 @@ function App() {
       <Notifications options={notifications} />
       <NavBar />
       <ProductsProvider>
-        <Routes>
-          <Route
-            path="/"
-            element={
-              <>
-                <SearchBar />
-                <Home />
-                <ProductIndex />
-              </>
-            }
-          />
-          <Route path="/productcategory/:id" element={<ProductCategory />} />
-          <Route path="/product/:id" element={<ProductScreen />} />
-        </Routes>
+        <CartProvider>
+          <Routes>
+            <Route
+              path="/"
+              element={
+                <>
+                  <SearchBar />
+                  <Home />
+                  <ProductIndex />
+                </>
```

Comment:
Maybe put these three sub components into a parent "HomeScreen" so it can be cleaner like the rest of the routes

Reply:
done